docs/products/kafka/concepts/partition-segments.rst

```diff
@@ -0,0 +1,65 @@
+Partition segments
+==================
+
+Apache Kafka divides topics partition data into **segment** files (with ``.log`` suffix) stored on the file system. Each segment file is named using the offset of the first message (a.k.a. **base offset**) contained. E.g. the segment file ``04.log`` contains the message with offset ``4`` as first entry. 
```

Comment:
```suggestion
Apache Kafka divides topics partition data into **segment** files (with ``.log`` suffix) stored on the file system. Each segment file is named using the offset of the first message (a.k.a. **base offset**) contained. For example, the segment file ``04.log`` contains the message with offset ``4`` as first entry. 
```
both are right, I just think looks better a non abbreviated version here as it is the beginning of a sentence but this is IMHO